internal/python/version.go

```diff
@@ -0,0 +1,49 @@
+package python
+
+import (
+	"log"
+
+	"github.com/Masterminds/semver/v3"
+)
+
+const defaultPython3Version = "3.8"
+
+// python3Versions is a list of all the Python3 versions.
+var python3Versions = []*semver.Version{
+	semver.MustParse("3.0"),
+	semver.MustParse("3.1"),
+	semver.MustParse("3.2"),
+	semver.MustParse("3.3"),
+	semver.MustParse("3.4"),
+	semver.MustParse("3.5"),
+	semver.MustParse("3.6"),
+	semver.MustParse("3.7"),
+	semver.MustParse("3.8"),
+	semver.MustParse("3.9"),
+	semver.MustParse("3.10"),
+	semver.MustParse("3.11"),
+	semver.MustParse("3.12"),
+}
```

Comment:
This array is better to be generated automatically, like what I do in `nodejs` planner, so we can update this field by running `go generate ./...` regularly. You can add a `TODO` for this.